surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+
+- Less than 3 months
+- 3 to 6 months
+- 1 year
+- 2-3 years
+- 3-5 years
+- 5+ years
+
+> **justification**
+>
+> Useful for cohort analysis, the problems faced by contributors may differ
+> substantially for new contributors vs long time maintainers.
+
+### How are you associated with the Rust Project?
+
+- I am a member of a Rust Team.
+- I am a lead of a Rust Team.
+- I am a member of a Rust working group or project group.
+- I am a lead of a Rust working group or project group.
+- I am not a member of a Rust Team, working group, or project group, but I
+  would like to be.
+- I am not a member of a Rust team, working group, or project group, and I have
+  no intention to join one.
+
+> **justification**
+>
+> Cohort analysis. Some of the subsequent questions are only relevant to
+> certain cohorts.
+
+### What kind of work do you do within the Rust Project?
+
+Select all that apply:
+
+- contribution work
+- maintainership work
+- reviews
+- coordination
+- organization
+
+> **justification**
+>
+> Useful for cohort analysis. Problems will likely differ based on the work
+> being done. Those contributing new code will likely face diff problems than
+> those who mainly review or coordinate work.
+
+### What sort of work would you like to be focused on?
+
+Free Form.
+
+> **justification**
+>
+> Useful in identifying how satisified our contributors are with the work
+> they're currently doing or if we should be helping them branch out into new
+> areas / achieve new goals.
+>
+> This was one of the best received questions during the pre-survey interviews
+> and seemed to elicited thoughtful answers.
+
+### My experience contributing to the Rust Project so far has been positive.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### I feel positive about the project leadership and governance structure.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel positive about my own leadership within the project.
+
+**Skip if not a leader of a team or group**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel my team/working group/project group is healthy.
+
+**Skip if not a member of the Rust Project.**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### Are you burnt out or do you feel you're at risk of burning out?
```

Comment:
Should we ask specifically about burn out related to contributing to Rust? Some people might feel burnt out due to their jobs or personal lives which would impact their contribution to Rust but feels distinct from burn out caused by contribution.

Reply:
Good point, and I think we should probably ask about both. If many of our maintainers are burnt out because of external factors that might still influence what kind of support we provide.